tck/src/main/java/org/reactivestreams/tck/IdentityProcessorVerification.java

```diff
@@ -738,6 +774,20 @@ public void expectNextElement(ManualSubscriber<T> sub, T expected) throws Interr
       }
     }
 
+    public void expectAnyNextElement(ManualSubscriber<T> sub, Collection<T> expected) throws InterruptedException {
```

Comment:
Since this is an API I'd rather have the types be specific.

```
private final <T extends Comparable<T>> Set<T> toSet(T... ts) {
   final Set<T> set;
   if (ts.length == 0) set = Collections.emptySet();
   else {
      set = new HashSet<T>();
      for(final T t : ts)
        set.add(t);
   }
   return set;
}
```